.eleventy.js

```diff
@@ -81,6 +81,19 @@ module.exports = function(eleventyConfig) {
         return postDate.isAfter(spacetime.today()) || postDate.isSame(spacetime.today())
     });
 
+    eleventyConfig.addFilter('countDays', (date) => {
```

Comment:
Do we want to use: https://momentjs.com/?

Reply:
not sure we need to include it given that we're already able to calculate date differences effectively?

Reply:
also `spacetime`z = 398kB, `Moment.js`z = 4MB